src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/ContentHelper.CoreClr.cs

```diff
@@ -37,11 +38,30 @@ internal static StringBuilder GetRawContentHeader(HttpResponseMessage response)
                 raw.AppendFormat("{0} {1} {2}", protocol, statusCode, statusDescription);
                 raw.AppendLine();
             }
+            
+            HttpHeaders[] headerCollections = 
+            {
+                response.Headers, 
+                response.Content == null ? null : response.Content.Headers
+            };
 
-            foreach (var entry in response.Headers)
+            foreach(var headerCollection in headerCollections)
```

Comment:
Minor point, insert a space after 'foreach' here and 'if' below to match the rest of the code.